tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php

```diff
@@ -179,7 +162,7 @@ private function getContainerBuilderProphecy()
 
         $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
         $containerBuilderProphecy->getParameter('kernel.bundles')->willReturn([
-            'DoctrineBundle' => 'Doctrine\Bundle\DoctrineBundle\DoctrineBundle',
+            'DoctrineBundle' => DoctrineBundle::class,
```

Comment:
There are other places where this is not changed. Perhaps we should make this consistent... (and for NelmioApiDocBundle as well)